xml/operations-maintenance-update_maintenance.xml

```diff
@@ -165,6 +165,20 @@
         </para>
        </listitem>
       </itemizedlist>
+     </step>
+     <step>
+      <para>
+       Demonstrate version changes by running <literal>hostnamectl</literal>
```

Comment:
Perhaps _confirm_ instead of _demonstrate_?

Reply:
confirm is better. fixed.